hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSEditLogAsync.java

```diff
@@ -117,6 +125,7 @@ void openForWrite(int layoutVersion) throws IOException {
   public void close() {
     super.close();
     stopSyncThread();
+    logSyncNotifyExecutor.shutdown();
```

Comment:
It could meet issue when transition active to standby then transition back here. Because executor has been shutdown and no response will be sent. Moreover I suspect that namenode process could fatal in this case. In my opinion, we do not need to shutdown this executor. FYI. Thanks.